### DISCUSSION TITLE
Plus view limited to today?

### DISCUSSION THREAD

**Discussion by jvallery** [blakeblackshear/frigate#14239] [Ask A Question]
I use the "Plus" view (previously snapshots) to get a glimpse of activity. It's nice to quickly scroll through all the snapshots with bounding boxes to get a summary of activity.

At some point recently (maybe 0.14.1?) there seems to have been a change where I can only scroll back to ~midnight. I can't scroll earlier into yesterday (or further back at all).  Previously it would dynamically load older images as I scrolled. There seemed to have been a bug where if a new snapshot came in while viewing the page it would jump back to the top. I think in the process of fixing that bug it has stopped loading older images?

Am I crazy?

**Comment by NickM-27** [comment]
The PR fixed scrolling in general which would never let you get past the first set of results. I just tested on the latest build and it is working fine for me.
